src/dispatch.yaml

```diff
@@ -1,6 +1,9 @@
 dispatch:
   - url: "*/api/*"
     service: py3-api
+  
+  - url: "*/clientapi/*"
+    service: py3-api
```

Comment:
reminder that this dispatch file is not used, the one in `master` is.

Though maybe we should copy that one here and deploy from this one.

Reply:
This is what I want for testing, where we don't have to do the default routes, but I can still run a local app build to change the service url directly to hit it on py3 and test it